GooglePlayInstant/Samples/TestApp/Editor/TestAppBuilder.cs

```diff
@@ -47,5 +49,21 @@ private static void ConfigureProject()
             PlayInstantBuildConfiguration.SetInstantBuildType();
             PlayerSettings.applicationIdentifier = BundleIdentifier;
         }
+
+        /// <summary>
+        /// Gets the apk path passed in via command line.
+        /// </summary>
+        private static string GetApkPath()
+        {
+            var args = System.Environment.GetCommandLineArgs();
+            for (var i=0; i<args.Length-1; i++)
+            {
+                if (Equals(args[i], ApkPathArg))
```

Comment:
Nit: prefer == for string equality in C# (it's not Java).

Reply:
Done